testsuites/CBLTester/CBL_Functional_tests/APITests_sc/test_basic_sc.py

```diff
@@ -0,0 +1,44 @@
+import random
+import pytest
+from keywords.utils import random_string
+
+
+@pytest.mark.usefixtures("class_init")
+class TestBasicSC(object):
+
+    def test_create_scope(self, scope_name):
+        ''' Create scope with different name '''
+        self.server_obj.create_scope(scope_name)
+
+    def test_create_collection(self, scope_name, collection_name):
+        '''
+        Test create collection
+        '''
+        self.server_obj.create_collection()
+
+    def test_create_scope_already_exist(self, scope_name):
```

Comment:
Maybe it's worth to have a cleanup at the end of the test, deleting the scope? It's generally a good practice, although maybe we have a different way to cleanup at the end of the test?

Reply:
This test to ensure that data in current collection will not be deleted if we create a new one with same name